services/ui-src/src/page/DescribeForms.tsx

```diff
@@ -0,0 +1,589 @@
+import React from "react";
```

Comment:
This whole component and its subcomponents are just brilliant......there is no way I could have done it this well. Bravo!